Base.rte/Devices/Tools/Medikit/Medikit.ini

```diff
@@ -0,0 +1,194 @@
+///////////////////////////////////////////////////////////////////////
+// Medikit
+
+
+AddEffect = MOPixel
+	PresetName = Heal Glow
+	Mass = 0.0
+	GlobalAccScalar = 0.0
+	AirResistance = 0.0
+	IgnoreTerrain = 1
+	LifeTime = 200
+	Sharpness = 0.0
+	HitsMOs = 0
+	GetsHitByMOs = 0
+	IgnoreTerrain = 1
+	Color = Color
+		R = 255
+		G = 0
+		B = 255
+	Atom = Atom
+		Material = Material
+			CopyOf = Air
+		TrailColor = Color
+			R = 255
+			G = 0
+			B = 255
+		TrailLength = 1
+	ScreenEffect = ContentFile
+		FilePath = Base.rte/Actors/Mecha/Medic/HealEffectGlow.bmp
+	EffectAlwaysShows = 1
+	EffectStartTime = -1
+	EffectStopTime = 200
+	EffectStartStrength = 0.0
+	EffectStopStrength = 0.5
+
+
+AddAmmo = Magazine
+	PresetName = Medikit Magazine
+	Mass = 0
+	HitsMOs = 0
+	GetsHitByMOs = 0
+	SpriteFile = ContentFile
+		FilePath = Base.rte/Null.bmp
+	FrameCount = 1
+	SpriteOffset = Vector
+		X = 0
+		Y = 0
+	EntryWound = AEmitter
+		CopyOf = Dent Metal
+	ExitWound = AEmitter
+		CopyOf = Dent Metal
+	AtomGroup = AtomGroup
+		AutoGenerate = 1
+		Material = Material
+			CopyOf = Bullet Metal
+		Resolution = 2
+		Depth = 0
+	DeepGroup = AtomGroup
+		AutoGenerate = 1
+		Material = Material
+			CopyOf = Bullet Metal
+		Resolution = 3
+		Depth = 1
+	DeepCheck = 0
+	JointStrength = 200
+	JointStiffness = 1
+	JointOffset = Vector
+		X = 0
+		Y = 0
+	DrawAfterParent = 1
+	RoundCount = 1
+	RTTRatio = 0
+	RegularRound = Round
+		CopyOf = Null Round
+	Discardable = 0
+
+
+AddDevice = HDFirearm
+	PresetName = Medikit
+	AddToGroup = Tools
+	Description = Healing supplies to-go! Heal your allies or yourself.
+	Mass = 2.0
+	HitsMOs = 0
+	GetsHitByMOs = 1
+	ScriptPath = Base.rte/Devices/Tools/Medikit/Medikit.lua
```

Comment:
Didn't think of it since there wasn't one already.

Reply:
So what is the verdict on this?
I do think it belongs in `Special`.

Reply:
I'd say we create a Special folder once there's something else that also belongs there?

Reply:
That will just lead to that folder never being created.
Create the folder, put the medkit in it, it will make it more obvious that stuff of the sorts goes into that folder.